README.md

```diff
@@ -12,7 +12,9 @@
 
 ## Intro video
 
-[![Claudie intro video](https://markdown-videos.deta.dev/youtube/q4xdAiHYxZQ)](https://youtu.be/q4xdAiHYxZQ)
+<p align="center">
+  <a href="https://youtu.be/q4xdAiHYxZQ"><img src="https://markdown-videos.deta.dev/youtube/q4xdAiHYxZQ)"></a>
```

Comment:
great!
one thing though, this does not render:
![image](https://user-images.githubusercontent.com/61180606/220602756-b9ed9a49-1a2a-4300-b8ef-367a94e45fd3.png)

there's an extra `)` at the end of `src`... @MarioUhrik

Reply:
fixed